src/views/camp/camp.jsx

```diff
@@ -0,0 +1,137 @@
+var React = require('react');
+var injectIntl = require('react-intl').injectIntl;
+var FormattedMessage = require('react-intl').FormattedMessage;
+var render = require('../../lib/render.jsx');
+var TitleBanner = require('../../components/title-banner/title-banner.jsx');
+var FlexRow = require('../../components/flex-row/flex-row.jsx');
+
+var Page = require('../../components/page/www/page.jsx');
+
+require('./camp.scss');
+
+var Downdeep = injectIntl(React.createClass({
+    type: 'Downdeep',
+    render: function () {
+        return (
+            <div>
+                <TitleBanner className="masthead mod-blue-bg">
+                    <img src="/images/downdeep/ocean-top.svg" alt="Top" className="topImg"/>
+                    <div className="inner">
+                        <div className="title-content">
+                            <h1 className="title-banner-h1">
+                                <FormattedMessage id='downdeep.title' />
```

Comment:
I think you should change the localization namespace to also be `camp` vs. `downdeep`.